### ISSUE TITLE
MSYS2 launcher error

### ISSUE THREAD

**Issue by HellJon6** [msys2/MSYS2-packages#3501]
### Description / Steps to reproduce the issue

just tried installing msys2 on a fresh windows 11 22H2 22621.1105 install, everything seems to work fine during MSYS2 installation and the terminal opens and works fine ONLY the first time, after I close it and try to open again it shows this message "could not find configuration file"(0x0)... I tried updating it, uninstalling and installing from safe mode but I always get the same error.
![2023-01-19 00 58 53](https://user-images.githubusercontent.com/123053530/213367426-0fb21050-6a17-4576-8287-ce984308aa06.png)

### Expected behavior

The terminal should open

### Actual behavior

the program is not opening, instead it shows a message saying "could not find configuration file"

### Verification

- [X] I have verified that my MSYS2 is up-to-date before submitting the report (see https://www.msys2.org/docs/updating/)

### Windows Version

MSYS_NT-10.0-22621

**Comment by Biswa96** [comment]
Please make sure mingw64.ini file exists in msys2 installation directory. Also if possible exclude msys2 installation directory in antivirus settings. Antivirus programs may delete files.

**Comment by HellJon6** [comment]
yes, the mingw64.ini file is in the installation directory. windows defender is completely disabled as well. I really have no idea what could be going on.

**Comment by lazka** [comment]
Did you install it to some special location, or the default `C:\msys64` ?

**Comment by HellJon6** [comment]
> Did you install it to some special location, or the default `C:\msys64` ?

Default location, I did try installing to different locations as well, including E drive, but it always throws me the same error.

**Comment by lazka** [comment]
No idea sadly. The only difference between the first time it opens from the installer and starting manually is that it starts `ucrt64.exe` there.

**Comment by HellJon6** [comment]
> No idea sadly. The only difference between the first time it opens from the installer and starting manually is that it starts `ucrt64.exe` in the installer.

yes,the first time It is installed it opens ucrt64.exe fine, but after closing it neither ucrt64, mingw64 or any other of the executables will open, they all show the same error

**Comment by yashpalgoyal1304** [comment]
maybe related: https://github.com/msys2/MSYS2-packages/issues/3102

**Comment by revelator** [comment]
does running the msys2_shell.cmd with the option -ucrt64 work ? if it does not then something interferes with the shell.
